Sources/Testing/Issues/Issue.swift

```diff
@@ -101,8 +100,12 @@ public struct Issue: Sendable {
   }
 
   /// The severity of this issue.
-  @_spi(Experimental)
   public var severity: Severity
+  
+  /// If the issues is a failing issue.
+  public var isFailure: Bool {
```

Comment:
I don't think this member is necessary; whether or not a warning is a failure might depend on external configuration (some equivalent to `-Werror` perhaps.)